src/networkpolicy/networkPolicy.go

```diff
@@ -1598,13 +1598,31 @@ func mergeIngressPolicies(existPolicy types.KnoxNetworkPolicy, policies []types.
 						}
 					}
 				}
+			} else if len(newIngress.FromCIDRs) > 0 && len(newIngress.ToPorts) > 0 {
```

Comment:
In this condition only `FromCIDRs` at index 0 is checked. What happens if we get multiple `FromCIDRs` in the rule.

Reply:
Each egress rule contains only one CIDRs. Hence no issue.